### ISSUE TITLE
Label creation, assignment to tickets broken after last update

### ISSUE THREAD

**Issue by schneidernicholas** [everyday-as/gmodstore-issues#1185]
### Expected Behavior

- When clicking "Create Label" the new label should be created.
- When assigning a label to a ticket, it should be added and persist after refreshing.

### Actual Behavior

- Clicking "Create Label" does nothing, except produce a 403 error in the console.
- Assigning a label to a ticket works in the UI, but after refreshing it does not persist, so it has not actually been added as a label to the ticket.

### Reproduction Steps

```markdown
1. Navigate to a customer's ticket.
2. Attempt to assign a label to the ticket.
3. Refresh the page to see that the label was not successfully persisted to the ticket.
4. Click the edit labels gear.
5. Click "New Label"
6. Specify a valid label name and color.
7. Click "Create label" and nothing will happen, returning a 403 in the console.
```

### Browsers

Edge (Chromium) 100.0.1185.39 (arm64)

### Example URL

https://www.gmodstore.com/dashboard/tickets/tags/thirdperson-an-advanced-third-person-suite/edit

### GmodStore Version

v11.0.11

### Additional Information

In addition to the example URL above, any addon customer tickets fail to have labels assigned to them.

403 error mentioned above:
![image](https://user-images.githubusercontent.com/14037442/163600889-6a7b1cd2-d9e4-44c4-af06-af7d355cbeb2.png)

This may need to be split into multiple issues, but it seems likely these two things are related since they both broke last update. Seems to be a larger overall issue with the label system and the new update.

**Comment by TheodoorP** [comment]
Fixed in the next release (v11.0.12)

**Comment by owainjones74** [comment]
@TheodoorP This issue seems to have come back for me ![](https://i.notgyazo.club/9OTuQBQVL.png)
